src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class Node<T> {
```

Comment:
lets make this class an inner class in MyLinkedList
because it should not be used outside that class